spec/Modules.tex

```diff
@@ -21,7 +22,11 @@ \section{Module Definitions}
 A module is declared with the following syntax:
 \begin{syntax}
 module-declaration-statement:
-  `module' module-identifier block-statement
+  privacy-specifier[OPT] `module' module-identifier block-statement
+
+privacy-specifier:
+  `private'
+  `public'
 
```

Comment:
Hm shouldn't 'privacy-specifier' be declared only once? This duplicates the rule in the Procedures chapter.